### ISSUE TITLE
MSA viewer doesn't deal with expanded window

### ISSUE THREAD

**Issue by olsonanl** [PATRIC3/patric3_website#1857]
If you expand the MSA viewer window the title doesn't update along with the data.

![screen shot 2018-01-22 at 4 05 27 pm](https://user-images.githubusercontent.com/2152226/35246936-2db2194e-ff8e-11e7-9b6d-4986cd255c9a.png)

**Comment by JoshuaVSherman** [comment]
@olsonanl Are you talking about the title on the browser tab?

**Comment by JoshuaVSherman** [comment]
* select Coxiella, select Features, select at least two features, on the green side icon menu, click MSA
* a new browser tab opens

**Comment by olsonanl** [comment]
No, if you bring up MSA, then widen the window, the MSA header with the large letters no longer spans the data displayed below it.

**Comment by JoshuaVSherman** [comment]
@olsonanl good morning, so you need to select MSA, amino acids to get that view, but i'm still not able to reproduce your issue, so I will discuss with @aswarren 
![image](https://user-images.githubusercontent.com/13340120/35512164-c37525aa-04cc-11e8-93f6-6dd3a9714d36.png)

**Comment by JoshuaVSherman** [comment]
@olsonanl hey Bob, can you please paste a link to your example with this issue, I spoke to Andrew, thanks

**Comment by olsonanl** [comment]
It's easy to reproduce in Safari anyway. Just bring up an MSA on a few genomes, expand the window, and scroll. See for instance

https://www.dropbox.com/s/wesjfyekfaplrt0/msa-bug.mov?dl=0
